packages/operations/health-check-cli/src/healthcheck.ts

```diff
@@ -0,0 +1,271 @@
+import Connector from "@walletconnect/core";
+import * as cryptoLib from "@walletconnect/iso-crypto";
```

Comment:
Did you have any issues using the `@walletconnect/client`? It would be great to replace it for these two libraries

Reply:
I did not try, but I can replace it with the client. Looks like it does the same job of few lines.